www/shared/styles/imports/custom-medias.css

```diff
@@ -4,6 +4,9 @@
    Use it like so: @media (--layout-small) { ... }
    ========================================================================== */
 
+/* this need to be according to the sum of --layout-container-width and 2 * --layout-edge-spacing-x */
+@custom-media --layout-lte-container-width (width <= 96.25em); /* 1540px */
```

Comment:
The `container` seems a bit generic.

We already have `edge-spacing` which means a safe area between the viewport and the actual "app content".  What if we name that container `layout-content`? This way, it would become:

`--layout-content-max-width`
`--layout-content-spacing-x`
`--layout-content-spacing-y`

and finally, this custom media would be `--layout-content-lte-max-width`.

Thoughts?

Reply:
I agree with all less the custom-media name.
I think should be `--layout-lte-content-max-width` to have consistency with the others custom-media.

What you think?

Reply:
Looks good!

Reply:
done! 👍